### PR TITLE
Add a news post discussing the C&C Remastered Collection and TS progress.

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

File: content/news/devblog-20200629.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200629-civilians.png" width="600" alt="Red Alert civilians">
+
+The full range of Red Alert civilians are now available thanks to details learned from the C&C Remastered Collection source code release.
+
+</div>
+
+One of the surprise announcements for the Remastered Collection was that EA would be open sourcing the dll files containing the gameplay logic (a slightly modified version of the original 90's source code) in order to support modding, and that it would be [released under the GPLv3](https://www.ea.com/games/command-and-conquer/command-and-conquer-remastered/news/remaster-update-modding) to ensure license compatibility with OpenRA and other community projects. We are extremely grateful to Jim and the team for making this happen, and proud to have played a part in this RTS milestone.
```
Comment (pchote) [inline content/news/devblog-20200629.md:19]: I'm not sure whether "surprise" or "exciting" works better here...

File: content/news/devblog-20200629.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The first of these are the isometric selection boxes used by structures. This may sound like a simple cosmetic change, but behind the scenes many changes were required to replace code designed for simple rectangular regions to work efficiently with more complicated shapes (such as the hexagons used by isometric buildings). The engine-level changes were actually added back in [release-20180218](/news/release-20180218/), but we have only recently had time to adapt the game code that defines and draws the selection boxes to make use of it. OpenRA's isometric selection boxes have some visual tweaks compared to the original game to better integrate with OpenRA's dynamic zooming and dynamic progress bars.
```
Comment (pchote) [inline content/news/devblog-20200629.md:36]: I'm not too happy with the wording of this paragraph, but don't yet have a specific idea on how to fix it.

File: content/news/devblog-20200629.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ <video width="600" autoplay loop>
+  <source src="/images/news/20200629-banshees.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video> 
+
+Tiberian Sun aircraft can finally pitch and bank while flying.
+
+</div>
+
+One of the most memorable aspects of Tiberian Sun was its ambience and aesthetics, which was in a large part created by its lighting effects. Work is almost complete on adding this to the OpenRA engine, with changes to smooth out harsh steps and apply the to units making them even more immersive.
```
Comment (pchote) [inline content/news/devblog-20200629.md:50]: Maybe drop aesthetics here... too many "a" words in a row.

**Comment by RobFog** [comment]
Pull request title: "Remasted" → "Remastered"

File: content/news/devblog-20200629.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+We believe that both approaches can coexist, and intend for OpenRA to continue following the same philosophies that have guided it so far. [Comments](https://forum.openra.net/viewtopic.php?p=309029#p309029) [from](https://old.reddit.com/r/commandandconquer/comments/9x1zcz/cc_remastered_announcement_from_ea/e9ozqrs/) [EA](https://old.reddit.com/r/Games/comments/gswgbw/amaa_were_the_team_behind_the_command_conquer/fs7ycc7/) suggest that they are also ok with the current situation, and will not interfere with OpenRA as it currently exists.
+
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200629-civilians.png" width="600" alt="Red Alert civilians">
+
+The full range of Red Alert civilians are now available thanks to details learned from the C&C Remastered Collection source code release.
```
Comment (pchote) [inline content/news/devblog-20200629.md:15]: s/are available/will be available in the next OpenRA release/

File: content/news/devblog-20200629.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+The next OpenRA release will contain some initial small fixes based on what we have already learned. One example of this is the original palette remap tables that the original Red Alert used to create the different civilian infantry types that were previously missing from OpenRA's version. There is a lot more to learn, and we expect that this knowlege will influence and inspire many details in the evolution and growth of the OpenRA games over the next few years.
+
+A popular question has been whether OpenRA will add support the remastered game assets, and I can now confirm (see [this GitHub comment](https://github.com/OpenRA/OpenRA/issues/18239#issuecomment-646380281) for details) that we will work towards this goal. This will take time, as there are a number of technical challenges that must first be solved. We will provide more information on these plans in future posts, or you can follow progress as we proceed on GitHub.
```
Comment (pchote) [inline content/news/devblog-20200629.md:25]: support *for* the

File: content/news/devblog-20200629.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+Tiberian Sun aircraft can finally pitch and bank while flying.
+
+</div>
+
+One of the most memorable aspects of Tiberian Sun was its ambience and aesthetics, which was in a large part created by its lighting effects. Work is almost complete on adding this to the OpenRA engine, with changes to smooth out harsh steps and apply the to units making them even more immersive.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200629-lighting.png" width="600" alt="Tiberian Sun lighting effects">
+
+Tiberian Sun's dynamic lighting effects have also been completed.
```
Comment (pchote) [inline content/news/devblog-20200629.md:55]: "will soon be"

**Comment by pchote** [comment]
Pushed another pass of wording changes, and I am now mostly happy with this.

File: content/news/devblog-20200629.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+A lot has happened in the Command and Conquer community recently surrounding the release of the C&C Remastered Collection, and many people have been asking about what this means for the future of OpenRA. In this post I would like to give a few comments on this topic, and show off some some updates on OpenRA's work-in-progress version of Tiberian Sun.
```
Comment (abcdefg30) [inline content/news/devblog-20200629.md:1]: `and show off some some`

File: content/news/devblog-20200629.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+OpenRA's game engine has been developed from scratch over many years using different programming languages and principles to the original C&Cs. This means that we cannot directly copy the source code, but will use it instead as a reference to learn what made the original games tick and to translate concepts and implementation details across where appropriate to improve OpenRA.
+
+The next OpenRA release will contain some initial small fixes based on what we have already learned. One example of this is the original palette remap tables that the original Red Alert used to create the different civilian infantry types that were previously missing from OpenRA's version. There is a lot more to learn, and we expect that this knowlege will influence and inspire many details in the evolution and growth of the OpenRA games over the next few years.
```
Comment (abcdefg30) [inline content/news/devblog-20200629.md:23]: `knowlege`

Comment (abcdefg30) [reply content/news/devblog-20200629.md:23]: Also `games` -> `mods` perhaps?

File: content/news/devblog-20200629.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200629-selection.png" width="600" alt="Tiberian Sun selection boxes">
+
+Structures in Tiberian Sun now use the correct isometric-style selection boxes.
+
+</div>
+
+In the meantime, work has continued on OpenRA's version of Tiberian Sun. Visible progress has been slow over the last couple of years while we focused on rebuilding parts of the low-level game engine to add the flexibility required to support the "gen2" games (Tiberian Sun, Red Alert 2, and community mods based on them), without breaking support for "gen1" games (Tiberian Dawn, Red Alert, Dune 2000 and existing community mods). This work is now starting to pay off, as we are able to leverage the added flexibilty to enable some long-awaited features.
```
Comment (abcdefg30) [inline content/news/devblog-20200629.md:34]: `flexibilty`

File: content/news/devblog-20200629.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+ <video width="600" autoplay loop>
+  <source src="/images/news/20200629-banshees.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video> 
+
+Tiberian Sun aircraft can finally pitch and bank while flying.
+
+</div>
+
+One of the most memorable aspects of Tiberian Sun was its ambience, which was largely created by its lighting effects. Work is almost complete on adding this to the OpenRA engine, which includes additional improvements to smooth out harsh steps and apply the to units making them even more immersive.
```
Comment (abcdefg30) [inline content/news/devblog-20200629.md:50]: `apply the to units`?

**Comment by pchote** [comment]
Updated.

**Pull request merged**